src/components/MediaPlayer/MediaPlayer.js

```diff
@@ -210,6 +210,7 @@ const MediaPlayer = ({ enableFileDownload = false, enablePIP = false }) => {
     poster: isVideo ? getPoster(manifest, canvasIndex) : null,
     controls: true,
     fluid: true,
+    language: "fr", // TODO:: fill this information from props
```

Comment:
Should this be switched to `en` or removed?

Reply:
Good catch! It should be `en` for now.